database/methods/reviews.js

```diff
@@ -1,17 +1,14 @@
 const ReviewModel = require('../models/reviews.js');
 const counterMethods = require('./counter.js');
 
-module.exports.addReview = async (review) => {
-  const counter = await counterMethods.incrementReviewSeq();
-  // eslint-disable-next-line no-param-reassign
-  review.review_id = (counter) ? counter.seq + 1 : 1;
-  return new Promise((resolve, reject) => {
-    ReviewModel.create(review, async (err, data) => {
-      if (err) {
-        await counterMethods.decrementReviewSeq();
-        reject(err);
-      }
-      resolve(data);
-    });
-  });
-};
+module.exports.addReview = (review) => new Promise((resolve, reject) => {
+  counterMethods.incrementReviewSeq()
+    .then((counter) => {
+      // eslint-disable-next-line no-param-reassign
+      review.review_id = (counter) ? counter.seq + 1 : 1;
+      resolve(ReviewModel.create(review));
+    })
+    .catch((err) => reject(err));
+});
+
+module.exports.getReviews = (options, limit = 0) => ReviewModel.find(options, '-_id -__v').limit(limit);
```

Comment:
Couple of structures most commonly used when exporting modules are (essentially to avoid duplication of 'module.exports'):
1:
module.exports = {
  addReview,
  getReviews,
};

or 2:

module.exports = {
  addReview: (review) => new Promise((resolve, reject) => {
    counterMethods.incrementReviewSeq()
      .then((counter) => {
        // eslint-disable-next-line no-param-reassign
        review.review_id = (counter) ? counter.seq + 1 : 1;
        resolve(ReviewModel.create(review));
      })
      .catch((err) => reject(err));
  }),
  getReviews: ((options, limit = 0) => ReviewModel.find(options, '-_id -__v').limit(limit)),
};

Reply:
Opted to option 1